PivovarAndrew/hometask-2/get_middle_character.rb

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def get_middle(str)
+  middle_symbol_point = str.length / 2 - 1
+  middle_symbol = str[middle_symbol_point]
+  middle_next_symbol = str[middle_symbol_point + 1]
```

Comment:
array can receive range as argument
`"test"[2..3]   # => "es"`
read thish please
https://ruby-doc.org/core-2.7.0/Array.html#method-i-5B-5D

Reply:
redundant variable `middle_next_symbol` if word length  is odd. remove

Reply:
why do you operate characters instead of numbers?? numbers is easier for CPU